docs/install/Knative-custom-install.md

```diff
@@ -304,23 +304,29 @@ commands below.
    completed until the upgrade process finishes.
 
 1. To install Knative components or plugins, specify the filenames in the
-   `kubectl apply` command:
+   `kubectl apply` command.
 
-   - To install an individual component or plgugin
+   - Run the `kubectl apply` command once with the
+   `-l knative.dev/crd-install=true` flag to install the CRDs first. (This
+   prevents race conditions, which cause intermittent install errors.) 
 
      ```bash
-     kubectl apply --filename [FILE_URL]
+     kubectl apply -l knative.dev/crd-install=true \
+     --filename [FILE_URL] \
+     --filename [FILE_URL]
      ```
 
-   - To install multiple components or plugins, append additional
-     `--filename [FILENAME]` flags to the `kubectl apply` command:
+   - Then run the `kubectl apply` command without the `-l` flag to complete the
+   install:
 
      ```bash
-     kubectl apply --filename [FILE_URL] --filename [FILE_URL] \
-       --filename [FILE_URL]
+     kubectl apply --filename [FILE_URL] \
+     --filename [FILE_URL]
```

Comment:
I was just trying to keep the formatting as similar as possible to the first version of the command. I wanted it to be visually very clear what has changed. And since, in the real versions of the commands, when [FILE_URL] is much longer, it makes sense to keep things on multiple lines.